crates/typstyle-core/src/pretty/config.rs

```diff
@@ -1,36 +1,36 @@
 /// Configuration Options for Typstyle Printer.
-#[derive(Debug, Clone)]
-pub struct PrinterConfig {
+#[derive(Debug, Clone, PartialEq, Eq, Hash)]
```

Comment:
Should we derive serde? Will typstyle finally have configuration files?

Reply:
We may add a feature flag to control this. Like `serde` will enable it. For config files, I'm not willing to add it at this moment.